q2_demux/_summarize/assets/src/plot.js

```diff
@@ -0,0 +1,229 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016-2017, QIIME 2 development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
+
+import * as d3 from 'd3';
+import addBox from './box';
+
+
+const plot = (data, props, container) => {
+  const plotContainer = d3.select(container);
+
+  const svg = plotContainer
+    .append('svg')
+      .attr('class', 'col-xs-12')
+      .style('display', 'block')
+      .style('margin', '0 auto')
+      .attr('width', props.width + props.margin.left + props.margin.right)
+      .attr('height', props.height + props.margin.top + props.margin.bottom);
+
+  const panel = plotContainer
+    .append('div')
+      .attr('class', 'col-xs-12')
+    .append('div')
+      .attr('class', 'panel panel-default');
+  panel.append('div')
+    .attr('class', 'panel-heading')
+    .html('Statistical Summary');
+  const table = panel.append('div')
+    .attr('class', 'stats')
+  .append('table')
+    .attr('class', 'table')
+    .style('margin-bottom', '0');
+
+  table
+    .append('thead')
+    .append('tr')
+      .selectAll('th')
+    .data([['Hover over a boxplot to learn more...', 9], ['', 3]])
+      .enter()
+    .append('th')
+    .text(d => d[0])
+    .attr('class', d => `col-xs-${d[1]}`)
+    .style('font-size', '10px');
+
+  const initialData = [
+    ['Position Number', '...'],
+    ['Minimum', '...'],
+    ['1st Quartile', '...'],
+    ['Median', '...'],
+    ['3rd Quartile', '...'],
+    ['Maximum', '...'],
+  ];
+
+  const rows = table
+    .append('tbody')
+      .selectAll('tr')
+      .data(initialData)
+    .enter()
+      .append('tr');
+
+  rows
+    .selectAll('td')
+    .data(d => d)
+      .enter()
+    .append('td')
+    .text(d => d);
+
+  const maxX = d3.max(data, d => d[0]);
+  const x0 = [0, maxX];
```

Comment:
I think there's an off-by-one error either where the box-plots are placed, or in the ticks placement. (It's easy to see if you zoom in enough to cause the ticks to be floats.)